Blish HUD/GameServices/ArcDps/Models/Ev.cs

```diff
@@ -1,36 +1,117 @@
+        /// </summary>
         public ushort DstInstId       { get; }
+        /// <summary>
+        /// If SrcAgent has a master (eg. minion, pet), this field will be equal to the map instance agent id of the master, zero otherwise.
+        /// </summary>
         public ushort SrcMasterInstId { get; }
+        /// <summary>
+        /// If DstAgent has a master (eg. minion, pet), this field will be equal to the map instance agent id of the master, zero otherwise.
+        /// </summary>
         public ushort DstMasterInstId { get; }
+        /// <summary>
+        /// Current affinity of SrcAgent and DstAgent (friend = 0, foe = 1 or unknown = 2).
+        /// </summary>
         public byte   Iff             { get; }
+        /// <summary>
+        /// TRUE if buff was applied, removed or damaging. Otherwise FALSE.
+        /// </summary>
         public bool   Buff            { get; }
+        /// <summary>
+        /// Physical Hit Result (normal hit = 0, was critical = 1, was glance =  2, was blocked = 3, was evaded = 4, interrupted the target = 5, was absorbed = 6, missed = 7, killed the target = 8, downned the target = 9).
+        /// </summary>
         public byte   Result          { get; }
+        /// <summary>
+        /// TRUE if the event is bound to the usage or cancellation of a skill (cast start to cast finish or cast cancel). Otherwise FALSE.
+        /// </summary>
         public bool   IsActivation    { get; }
+        /// <summary>
+        /// TRUE if buff was removed (for strips/cleanses: SrcAgent = relevant, DstAgent = caused it). Otherwise FALSE.
+        /// </summary>
         public bool   IsBuffRemove    { get; }
+        /// <summary>
+        /// TRUE if SrcAgent is above 90% health. Otherwise FALSE.
+        /// </summary>
         public bool   IsNinety        { get; }
+        /// <summary>
+        /// TRUE if DstAgent is below 50% health. Otherwise FALSE.
+        /// </summary>
         public bool   IsFifty         { get; }
+        /// <summary>
+        /// TRUE if SrcAgent is moving at time of event. Otherwise FALSE.
+        /// </summary>
         public bool   IsMoving        { get; }
+        /// <summary>
+        /// TRUE if a state change occured (SrcAgent is now alive, dead, downed and other ambigious stuff ex. when SrcAgent is self, DstAgent is a reward id and Value is a reward type such as a wiggly box.). Otherwise FALSE.
+        /// </summary>
         public bool   IsStateChange   { get; }
+        /// <summary>
+        /// TRUE if SrcAgent is flanking DstAgent at time of event. Otherwise FALSE.
+        /// </summary>
         public bool   IsFlanking      { get; }
+        /// <summary>
+        /// TRUE if all or part of damage was VS. barrier/shield. Otherwise FALSE.
+        /// </summary>
         public bool   IsShields       { get; }
+        /// <summary>
+        /// FALSE if buff damage happened during tick. Otherwise TRUE.
```

Comment:
Minor nitpick: All the other summaries start with what's up when the field is true, this is the only one inverted. Would be nice to have consistency there.